docker/mainnet/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   bodhi-server-mainnet:
     hostname: bodhi-server-mainnet
     container_name: bodhi-server-mainnet
-    restart: unless-stopped
+    restart: always
```

Comment:
Remind me why we had `unless-stopped` at the first place?

Reply:
ya it wasnt the proper restart policy to begin with. generally we want `always` in all cases to have max uptime.